api/src/main/kotlin/edu/wgu/osmt/security/SecurityConfig.kt

```diff
@@ -90,7 +91,7 @@ class SecurityConfig : WebSecurityConfigurerAdapter() {
             .mvcMatchers(GET, COLLECTION_DETAIL).permitAll()
 
             .mvcMatchers(POST, COLLECTION_SKILLS).permitAll()
-            .mvcMatchers(GET, COLLECTION_CSV).permitAll()
+            .mvcMatchers(GET, COLLECTION_CSV).hasRole(appConfig.roleAdmin)
```

Comment:
This will prevent all non-admins from downloading any collection CSV files. This seems like an unintended side effect, when the ask is that only admins download Draft collection CSVs.

Reply:
removed and placed check in endpoint body